en/operations/docker-containers.html

```diff
@@ -33,7 +33,32 @@ <h2 id="mounting-persistent-volumes">Mounting persistent volumes</h2>
   --volume $VESPA_LOG_STORAGE:/opt/vespa/logs --publish 8080:8080 vespaengine/vespa
 </pre>
 
+<h2 id="">Start Vespa container with Vespa user</h2>
 
+<p>
+  You can start the container directly as the `vespa` user. The `vespa` user and group within the container are
+  configured with user id `1000` and group id `1000`. The vespa user and group must be the owner of the `var` and
```

Comment:
`/opt/vespa/var`  and `/opt/vespa/logs` ?

Reply:
agree, fixed.